### DISCUSSION TITLE
How to Contribute?

### DISCUSSION THREAD

**Discussion by hrynevychroman** [vuejs/devtools#175] [Q&A]
I receive error with electron package, that error is gone now, but one more receiving, and don't know how to fixed that?

Change this 👇 
<img width="1455" alt="image" src="https://github.com/vuejs/devtools-next/assets/82209198/b509c9fc-c506-423c-aa16-ce967d7b90fa">

Started receiving error with alias configuration in `/client` package, but everything looks normal 🤔 
<img width="1689" alt="image" src="https://github.com/vuejs/devtools-next/assets/82209198/3d7e3af6-a7f0-48b7-b6c9-66001f5ea8ec">
<img width="1688" alt="image" src="https://github.com/vuejs/devtools-next/assets/82209198/976ad0de-6ebb-4fe9-ad1f-12dd60219b20">

Maybe someone can help to plug&play with forks 🙂

**Comment by webfansplz** [comment]
See https://devtools-next.vuejs.org/guide/contributing (The guide may not be detailed enough now, but I'll refine it later

**Reply by hrynevychroman** [reply]
There are compiling errors after run `pnpm dev`, don't think that it is ok 😅
